receiver/prometheusreceiver/internal/metricsbuilder_test.go

```diff
@@ -90,7 +90,7 @@ func runBuilderTests(t *testing.T, tests []buildTestData) {
 			mc := newMockMetadataCache(testMetadata)
 			st := startTs
 			for i, page := range tt.inputs {
-				b := newMetricBuilder(mc, true, testLogger)
+				b := newMetricBuilder(mc, true, "", testLogger)
```

Comment:
Please also add a test case for metrics builder where regex is not empty to cover the `matchStartTimeMetric` function.

Reply:
Done.